lib/rules/color-named/colord-utils.js

```diff
@@ -0,0 +1,75 @@
+const { colord, extend } = require('colord');
```

Comment:
[trivial] For consistency in the project, the file name `colordUtils.js` may be better than `colord-utils.js`.